### PR TITLE
Update mast-fish.md

### PR DESCRIPTION
**Pull request by pjoh6**

### CODE REVIEW

File: annotations/mast-fish.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+**_mast fish_** (English); _mastefisk_ (Danish); _Mastfisch_ (German)
+
+_**klofi** m., pl. klofar_ (Old Norse) [citations: [prose](https://onp.ku.dk/onp/onp.php?o44117)/[poetry](https://lexiconpoeticum.org/m.php?p=lemma&i=45381)]
+
+  A piece of wood that was placed on the keelson (ON _kerling_) to keep the mast upright and absorb stress while the ship was under sail. The mast fish was typically ten feet long and faced aft to help while the mast (ON _siglutré_, _sigla_) was raised (National Geographic).
```
Comment (apjmason) [inline annotations/mast-fish.md:5]:
```suggestion
  A large mass of wood that was placed on the keelson (ON _kerling_) to keep the mast upright and absorb stress while the ship was under sail. 
```

File: annotations/mast-fish.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+<div align="center">
+  
+  ![mast fish from Gokstad ship](../images/Kerling_00_Gokstad.png)  
+  Mast fish from the Gokstad ship (Nicolaysen Pl. I, Fig. 4). The mast fish sits above the keelson.
+
+</div>
+ 
+This feature is named for its fish-like shape, though it is sometimes also called the mast support or mast-partner (Nicolaysen).
```
Comment (apjmason) [inline annotations/mast-fish.md:14]:
```suggestion
The mast fish was typically ten feet long and faced aft to help while the mast (ON _siglutré_, _sigla_) was raised (National Geographic). This feature is named for its fish-like shape, though it is sometimes also called the mast support or mast-partner (Nicolaysen). 
```

Comment (apjmason) [reply annotations/mast-fish.md:14]: Moving this detail below the image to accord with other annotations

File: annotations/mast-fish.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+**_mast fish_** (English); _mastefisk_ (Danish); _Mastfisch_ (German)
+
+_**klofi** m., pl. klofar_ (Old Norse) [citations: [prose](https://onp.ku.dk/onp/onp.php?o44117)/[poetry](https://lexiconpoeticum.org/m.php?p=lemma&i=45381)]
+
+  A piece of wood that was placed on the keelson (ON _kerling_) to keep the mast upright and absorb stress while the ship was under sail. The mast fish was typically ten feet long and faced aft to help while the mast (ON _siglutré_, _sigla_) was raised (National Geographic).
+
+<div align="center">
+  
+  ![mast fish from Gokstad ship](../images/Kerling_00_Gokstad.png)
```
Comment (apjmason) [inline annotations/mast-fish.md:9]: Good call reusing this image!

File: annotations/mast-fish.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+</div>
+ 
+This feature is named for its fish-like shape, though it is sometimes also called the mast support or mast-partner (Nicolaysen).
+
+---
+
+  Nicolaysen, N. (1882). Langskibet fra Gokstad ved Sandefjord. Kristiania.
+
+  Paasche, Knut. "The Tune Viking Ship Reconsidered." _International Journal of Nautical Archaeology_ 49, no. 1 (2020): 29-48. https://doi.org/10.1111/1095-9270.12391.
```
Comment (apjmason) [inline annotations/mast-fish.md:20]: This and the Unger are not cited here. Are they needed?

**Review by apjmason — CHANGES_REQUESTED** [review]
This looks good Parker! I made a few edits you can commit with the Commit Suggestion button if approved. 

I suggest removing the unreferenced sources from this annotation

File: annotations/mast-fish.md (hunk 4 above)
Comment (pjoh6) [reply annotations/mast-fish.md]: I had kept these from the previous version of this annotation, but you're right--they aren't cited anywhere. I'll delete them.

**Review by apjmason — APPROVED** [review]
Looks good!

**Pull request merged**